integration/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,426 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /SUCCESS/}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Create an inventory and check database" do
+        subject do
+            command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"")
+        end
+
+        before(:context) do
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
+        end
+
+        # its("stdout") { should match "2" }
+        its("stderr") { should eq ""}
```

Comment:
Why checking on `stderr` here?

Reply:
This is a way to have least number of assertions while still ensuring the test case is properly checked. If there are no errors that means all the assertions are correct.

Reply:
Elaborating on that - `stderr` ensure there are no errors in the standard output. Database check confirms this.